docs/_assets/js/custom.js

```diff
@@ -1,64 +1,61 @@
 (function () {
   'use strict';
 
+  var $navBar = $('#home > .navbar');
+
   $(window).scroll(function () {
     var top = $(document).scrollTop();
+
     if (top > 50) {
-      $('#home > .navbar').removeClass('navbar-transparent');
+      $navBar.removeClass('navbar-transparent');
     } else {
-      $('#home > .navbar').addClass('navbar-transparent');
+      $navBar.addClass('navbar-transparent');
     }
-  })
+  });
 
   $('a[href="#"]').click(function (event) {
     event.preventDefault();
-  })
+  });
 
-  $('.bs-component').each(function () {
-    var $component = $(this);
-    var $button = $('<button class="source-button btn btn-primary btn-xs" role="button" tabindex="0">&lt; &gt;</button>');
+  $('.bs-component').each(function (i, element) {
+    var $component = $(element);
+    var $button = $('<button class="source-button btn btn-primary btn-xs" type="button" tabindex="0">&lt; &gt;</button>');
```

Comment:
`role="button` is useless with a `button`.

Unsure what `tabindex="0"` is supposed to fix?

Reply:
I'm unsure myself — changes look good to me.